auth/presentation/src/main/java/com/sergius/auth/presentation/login/LoginEvent.kt

```diff
@@ -0,0 +1,8 @@
+package com.sergius.auth.presentation.login
+
+import com.sergius.core.presentation.ui.UiText
+
+sealed interface LoginEvent {
+    data class Error(val error: UiText): LoginEvent
+    data object LoginSuccess: LoginEvent
```

Comment:
Minor: When having such enums/sealed hierarchies, I often see people duplicate information. The name of your interface `LoginEvent` already contains the information that all events in here relate to login, so `LoginEvent.LoginSuccess` would be duplicated. It could be `LoginEvent.Success` and would say the same :)